core-plugins/src/main/java/io/cdap/plugin/batch/joiner/Joiner.java

```diff
@@ -361,6 +362,13 @@ Schema getOutputSchema(Map<String, Schema> inputSchemas, FailureCollector collec
 
     return outputFieldInfo.values();
   }
+  private boolean inputSchemasAvailable(Map<String, Schema> inputSchemas) {
```

Comment:
style: should have a newline separating methods

Reply:
code moved to JoinerConfig.